### PR TITLE
Report ajax warnings to sentry

### PR DESCRIPTION
**Pull request by naphets123**

### CODE REVIEW

**Comment by naphets123** [comment]
[Report warnings/errors from desktop ajax requests to sentry](https://trello.com/c/fczZBBT9/424-report-warnings-errors-from-desktop-ajax-requests-to-sentry)

File: library/ajax/reset.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
         $message = GENERIC_LOGIN_ERROR;
         $redirect = '';
         logfile('Poging tot wachtwoord reset door onbekende gebruiker '.$_POST['email']);
```
Comment (HaGuesto) [inline library/ajax/reset.php]: Could you change the message for the logfile?

File: library/ajax/testdbdelete.php [hunk 2]
```diff
@@ -8,7 +8,9 @@
 
     // Test if user is a testusers and the database is a dev database
     if (!(in_array($settings['db_database'], $devdbs) && in_array($_SESSION['user']['email'], $testusers))) {
-        $return = ['success' => false, 'message' => 'You do not have access to delete test data!'];
+        $msg = 'You do not have access to delete test data!';
+        $return = ['success' => false, 'message' => $msg];
+        trigger_error($msg);
```
Comment (HaGuesto) [inline library/ajax/testdbdelete.php]: Access error should be a higher error_level. Best `E_USER_ERROR`

File: library/ajax/testdbdelete.php [hunk 3]
```diff
@@ -54,6 +56,7 @@
                     $return = 'true';
                 } else {
                     $return = 'false';
+                    trigger_error('No permission to delete this data');
```
Comment (HaGuesto) [inline library/ajax/testdbdelete.php]: Same as above.

File: library/ajax/reset.php (hunk 1 above)
Comment (naphets123) [reply library/ajax/reset.php]: Done, also changed another dutch warning i found in this file

File: library/ajax/testdbdelete.php (hunk 2 above)
Comment (naphets123) [reply library/ajax/testdbdelete.php]: Done

File: library/ajax/testdbdelete.php (hunk 3 above)
Comment (naphets123) [reply library/ajax/testdbdelete.php]: Done

**Pull request merged**